public/js/login.js

```diff
@@ -0,0 +1,71 @@
+import {
+  inputEnabled,
+  setDiv,
+  token,
+  message,
+  enableInput,
+  setToken,
+} from "./index.js";
+import { showLoginRegister } from "./loginRegister.js";
+import { showFoodItems } from "./food-items.js";
+
+let loginDiv = null;
+let email = null;
+let password = null;
+
+export const handleLogin = () => {
+  loginDiv = document.getElementById("logon-div");
```

Comment:
Since everything seems to work well, I think I'll leave some suggestions for the future: document.getElementById() technically returns either an Element object, _or null_. If the browser doesn't find logon-div for any reason, this function will break when it tries to add an event listener to null. For maximum safety, if you are getting a variable that might be null, you'll want to do something like `if (loginDiv == null) { console.error("loginDiv was null"); return; }`.

Your work is still 100% correct, no fixes are needed. But since you've completed this assignment flawlessly, I figured I would leave some tips for the future.